### PR TITLE
Adding projectionsto ServiceFrame

### PR DESCRIPTION
**Pull request by ue71603**
Based on discussion here: #537 

Pls check what you think.

### CODE REVIEW

File: examples/functions/patterns/Netex_KBIC_ParisNetwork_1.xml [hunk 1]
```diff
@@ -535,6 +535,18 @@ Includes the following:
 							</pointsInSequence>
 						</Route>
 					</routes>
+					<projections>
+						<LinkProjection version="any" id="tim:LP_TL_TP001o_TP4toTP5_PJ01" order="1">
+							<Name>Projection  mid way along link TL 4-5::&gt;RL 8-9 x </Name>
```
Comment (skinkie) [inline examples/functions/patterns/Netex_KBIC_ParisNetwork_1.xml:540]:
```suggestion
							<Name>Projection  mid way along link TL 4-5::&gt;RL 8-9 x</Name>
```

File: xsd/netex_part_2/part2_frames/netex_serviceFrame_version.xsd [hunk 2]
```diff
@@ -132,6 +132,11 @@ Rail transport, Roads and Road transport
 			<xsd:group ref="RouteInFrameGroup"/>
 			<xsd:group ref="FlexibleRouteInFrameGroup"/>
 			<xsd:group ref="SectionInFrameGroup"/>
+			<xsd:element name="projections" type="projections_RelStructure" minOccurs="0">
```
Comment (skinkie) [inline xsd/netex_part_2/part2_frames/netex_serviceFrame_version.xsd:135]: Can we make this uniform with the rest, hence a group?

Comment (ue71603) [reply xsd/netex_part_2/part2_frames/netex_serviceFrame_version.xsd:135]: done

**Review by skinkie — APPROVED** [review]

**Review by trurlurl — APPROVED** [review]

**Comment by nick-knowles** [comment]
This also required a change to to  Transmodel to add an abstract  spertye PROJECTIOn which edidnt previousy exist. This can the be referenced from SERVCIE FRAME
UML Documentation Transmodel
image

image

UML DOCUMENTATION NeTEx
image

**Pull request merged**